docs/blog/introduction-0.15.0/index.qmd

```diff
@@ -0,0 +1,202 @@
+    .with_columns(name=pl.concat_str(pl.col("name_given") + " " + pl.col("name_family")))
+    .fill_null(value="")
+    .select(["country", "name", "address", "city", "state_prov", "postcode"])
+)
+
+(
+    GT(peeps_mini)
+    .tab_header(title="Our Contacts (Born in 1988)")
+    .fmt_flag(columns="country")
+    .opt_vertical_padding(scale=0.5)
+    .cols_label(
+        country="",
+        name="Name",
+        address="Address",
+        city="City",
+        state_prov="State/Prov.",
+        postcode="Zip/Postcode",
+    )
+)
+```
+
+This slice of the `peeps` dataset has country codes in their 3-letter form (i.e., `"USA"`, `"SVN"`, and `"CAN"`) within the `country` column. So long as they are correct, `fmt_flag()` will perform the conversion to flag icons. Also, there's a little bit of interactivity here: when hovering over a flag, the country name will appear as a tooltip!
+
+We have the power to display multiple flag icons within a single cell. To make this happen, the country codes need to be combined in a single string where each code is separated by a comma (e.g., `"US,DE,GB"`). Here's an example that uses a portion of the `films` dataset:
+
+```{python}
+from great_tables import GT, google_font
+from great_tables.data import films
+import polars as pl
+
+films_mini = (
+    pl.from_pandas(films)
+    .filter(pl.col("director") == "Michael Haneke")
+    .with_columns(
+        title=pl.concat_str(pl.col("title") + " (" + pl.col("year").cast(pl.String) + ")")
+    )
+    .select(["title", "run_time", "countries_of_origin"])
+)
+
+(
+    GT(films_mini)
+    .fmt_flag(columns="countries_of_origin")
+    .tab_header(title="In Competition Films by the Michael Haneke")
+    .opt_stylize()
+    .tab_options(column_labels_hidden=True)
+    .opt_table_font(font=google_font("PT Sans"))
+)
+```
+
+The column `countries_of_origin` has these combined strings for each of the co-production films, where countries are arranged by decreasing level of contribution (e.g., `"FR,AT,RO,DE"` in the second row). The `fmt_flag()` method parses these strings into a sequence of flag icons that are displayed in the order provided. Each of the flags is separated by a space character but you can always change that default separator with the `sep=` argument.
+
+### Using `fmt_icon()` to include Font Awesome icons
+
+The new `fmt_icon()` method gives you the ability to easily include FontAwesome icons in a table. It uses a similar input/output scheme as with `fmt_flag()`: provide the *short* icon name (e.g., `"table"`, `"music"`, `"globe"`, etc.) or a comma-separated list of them, and `fmt_icon()` will provide the Font Awesome icon in place. Let's see it in action with an example that uses the `metro` dataset:
+
+```{python}
+from great_tables import GT
+from great_tables.data import metro
+import polars as pl
+
+metro_mini = (
+    pl.from_pandas(metro)
+    .tail(10)
+    .with_columns(
+        lines=pl.lit("train"),
+        connect_tramway = (
```

Comment:
I would define the services column here, instead of separately below. (So replace `connect_tramway =` with `services=`)

Something like removing the need for `lines=`, and just using...

```python
    .with_columns(
        services = (
            pl.when(pl.col("connect_tramway").is_not_null())
            .then(pl.lit("train, train-tram"))
            .otherwise("train")
        )
    )
```

Reply:
Great idea. Now fixed!